eo-runtime/pom.xml

```diff
@@ -34,11 +34,17 @@ SOFTWARE.
   <name>eo-runtime</name>
   <description>EO runtime library</description>
   <dependencies>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <version>5.8.1</version>
+      <scope>test</scope>
+    </dependency>
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>jul-to-slf4j</artifactId>
       <version>2.0.0-alpha5</version>
-      <scope>test</scope>
+      <scope>compile</scope>
```

Comment:
@marceloamadeu still wrong. We shouldn't force our users to use SLF4J. We just use JUL and _they_ decide later what bridge to use.

Reply:
@yegor256 I didn't understand how to do it. Please check that it is correct now.